### ISSUE TITLE
Add Hebrew morphological tagging via files from openscriptures

### ISSUE THREAD

**Issue by jomortal** [digitalbiblesociety/browserbible#22]
Was wondering if you had plans to add Hebrew morphological tagging. I'm sure you're aware of these projects, and realize the work that goes into this. You have one of if not the best free online Bible study tools. Love your work.

Please consider incorporating from one of these projects....
https://github.com/openscriptures/morphhb
https://github.com/eliranwong/OpenHebrewBible

**Comment by KenBitgood** [comment]
Hey JoMortal,
Thank you so much for the links and the encouragement.
Yes, we would love to see BibleBrowser (inScript) improved and updated.
Please pray for us that this would become a reality.
The possibilities are endless, the laborers are few right now.
Are you a Hebrew student?

Ken Bitgood
CEO, Digital Bible Society

**Comment by jomortal** [comment]
Yes, I am a pastor at a small church in Chicago. I graduated from Moody in 2015 with an emphasis in Biblical Languages. I keep up on them as best I can, and dabble in coding. I'm always looking for better ways to equip and educate the church, especially creating tools that would be beneficial. I've also had some work in the Bible translation world. I'd love to chat sometime. I hear what you're saying. It is hard to find others to do this type of labor with. Feel free to email me at jonathan@rogersparkchurch.com
